recipes/lunasvg/all/test_v1_package/CMakeLists.txt

```diff
@@ -8,4 +8,14 @@ find_package(lunasvg REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} ../test_package/test_package.cpp)
 target_link_libraries(${PROJECT_NAME} PRIVATE lunasvg::lunasvg)
-target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_14)
+if (lunasvg_VERSION VERSION_LESS_EQUAL "2.3.2")
```

Comment:
I think this cxx version logic is wrong - 3.0.0 should have c++17 right?

Reply:
@AbrilRBS 
Thank you for your review!
I fixed these conditions.